tests/harness/executor/cases.go

```diff
@@ -784,6 +784,11 @@ var stringCases = []TestCase{
 	{"string - contains - invalid", &cases.StringContains{Val: "candy bazs"}, false},
 	{"string - contains - invalid (case-sensitive)", &cases.StringContains{Val: "Candy Bars"}, false},
 
+	{"string - not contains - valid", &cases.StringNotContains{Val: "candy bars"}, false},
+	{"string - not contains - valid (only)", &cases.StringNotContains{Val: "bar"}, false},
+	{"string - not contains - invalid", &cases.StringNotContains{Val: "candy bazs"}, true},
+	{"string - not contains - invalid (case-sensitive)", &cases.StringNotContains{Val: "Candy Bars"}, true},
```

Comment:
you're identifying the first two tests as "valid" but then setting their validity to `false`, and vice-versa for the last two. Either the labels are wrong, or the implementation is backwards.

Reply:
Oops, thank you so much for catching that. I didn't catch the mislabeling.